### ISSUE TITLE
BreadcrumbItem component places `href` attribute on `li` element instead of `a`

### ISSUE THREAD

**Issue by nkrisc** [yesvelte/yesvelte#310]
**Describe the bug**
The BreadcrumItem component places its `href` prop as an `href` attribute on the `li` element instead of the `a` element. Because of this, the links do not work.

**To Reproduce**
For example, the following:

```
<BreadcrumbItem href="/">Home</BreadcrumItem>
```

Is rendered as:

```
<li href="/" aria-current="page" id="breadcrumb-item_2" class="y-el y-breadcrumb-item">
    <a id="breadcrumb-inner_3" class="y-el y-breadcrumb-inner">Home</a>
</li>
```

**Expected behavior**
The `href` attribute is expected to be on the anchor element, not the list item.

**Additional context**
Using yesvelte 0.0.2-next.93

**Comment by TheHadiAhmadi** [comment]
thanks @nkrisc for the report. we fixed it in `yesvelte@0.0.2-next.99`. here is the working REPL https://svelte.dev/repl/9dbdd72d4a2043aba11cf73ff1675f0d?version=3.55.1

**Comment by nkrisc** [comment]
Updated to 0.0.2-next.99 and it works just as expected.
